assets/helpers/types/option.js

```diff
@@ -0,0 +1,17 @@
+// @flow
+
+// ----- Type ----- //
+
+export type Option<A> = A | null;
```

Comment:
Is it potentially confusing that this `Option` is not really very much like Scalas?
I think I might expect to be able to map over it for instance.

Reply:
Fair point. I introduced this because I was writing `A | null` a lot and I wanted to formalise a type to handle that. I went with `Option` because I thought people who have seen that in Scala would be familiar with it. However, if you think it might cause confusion there are some alternatives:

1. Use a different name, e.g. `Maybe` which is used in Haskell. This is also the name Flow gives to `?A`; I don't know whether that would be more or less confusing.

2. Write a couple of helpers to bring the functionality closer to Scala, starting with `map`.

Reply:
Happy to go with it as is for now and add in helpers when they become useful. It would be great to try this out as well: https://medium.com/inside-rimeto/optional-chaining-in-typescript-622c3121f99b
but obviously requires TS